docs/en/enterprise-edition/rn/look-ahead-planned-updates-prisma-cloud/look-ahead-secure-the-infrastructure.adoc

```diff
@@ -216,101 +151,111 @@ config from cloud.resource where cloud.type = 'gcp' AND api.name = 'gcloud-armor
 |*Service*
 |*API Details*
 
-|*AWS Batch*
-//RLP-122581
+|*AWS Glue*
+
+tt:[*24.3.2*]
```

Comment:
No need to add tt:[*24.3.2*] tags in LA. Please remove them from all the ingestions listed in the table.